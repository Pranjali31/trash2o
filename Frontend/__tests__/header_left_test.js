import React from 'react';
import { render, fireEvent, waitFor } from '@testing-library/react-native';
import { NavigationContainer } from '@react-navigation/native';
import HeaderLeft from '../src/components/atoms/headerLeft/index';

test("index Component Loading", () => {
    render(<HeaderLeft  onPress={() => { }}  />)

})
test("index component snapshot", () => {
    let component = render(<HeaderLeft  onPress={() => { }}  />).toJSON();
    expect(component).toMatchSnapshot();
})