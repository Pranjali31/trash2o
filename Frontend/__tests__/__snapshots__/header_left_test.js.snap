// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`index component snapshot 1`] = `
<View
  accessible={true}
  collapsable={false}
  focusable={true}
  onClick={[Function]}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    Object {
      "marginHorizontal": 10,
      "opacity": 1,
    }
  }
>
  <View
    style={
      Object {
        "overflow": "hidden",
      }
    }
    testID="RNE__ICON__CONTAINER"
  >
    <View
      accessibilityRole="button"
      android_ripple={
        Object {
          "borderless": false,
          "color": "rgba(0, 0, 0, 0.3)",
          "radius": -5,
        }
      }
      disabled={false}
      testID="RNE__ICON__CONTAINER_ACTION"
    >
      <View
        style={
          Object {
            "alignItems": "center",
            "backgroundColor": "transparent",
            "justifyContent": "center",
          }
        }
        testID="RNE__ICON"
      >
        <Text
          allowFontScaling={false}
          selectable={false}
          style={
            Array [
              Object {
                "color": "white",
                "fontSize": 30,
              },
              Object {
                "backgroundColor": "transparent",
              },
              Object {
                "fontFamily": "Feather",
                "fontStyle": "normal",
                "fontWeight": "normal",
              },
              Object {},
            ]
          }
          testID="RNE__ICON__Component"
        >
          ï†œ
        </Text>
      </View>
    </View>
  </View>
</View>
`;
